BitmarkRegistry/Controllers/Main/SuspendedViewController.swift

```diff
@@ -0,0 +1,43 @@
+//
+//  SuspendedViewController.swift
+//  BitmarkRegistry
+//
+//  Created by Macintosh on 7/19/19.
+//  Copyright © 2019 thuyentruong. All rights reserved.
+//
+
+import UIKit
+
+// block user screen when there are critical issue need to be done firstly
+class SuspendedViewController: UIViewController {
+  // MARK: - Init
+  override func viewDidLoad() {
+    super.viewDidLoad()
+
+    setupViews()
+  }
+}
+
+// MARK: - Setup Views
+// TODO: Update Design UI
+extension SuspendedViewController {
+  fileprivate func setupViews() {
+    view.backgroundColor = .white
+
+    // *** Setup subviews ***
+    let label = CommonUI.descriptionLabel(text: "There are critical issue happening. We are checking and come back to you soon.")
```

Comment:
It's probably ok, but we should leave this in TODO list to polish the design and behavior of this screen.

Reply:
Would be better if we allow users to report the issue by intercom later from this screen.

Reply:
I did add into TODO list in line 22 for the UI design

Reply:
ok, thanks.
I will note the intercom implementation and mention it when we design more UI for this screen. :)